src/module/entities/TwodsixActor.ts

```diff
@@ -50,6 +50,27 @@ export default class TwodsixActor extends Actor {
       characteristic.current = characteristic.value - characteristic.damage;
       characteristic.mod = calcModFor(characteristic.current);
     }
+    const actorSkills = actorData.items.filter(
+      item => item.type === "skills"
+    ).map(
+      skill => [TwodsixItem.simplifySkillName(skill.name), skill.data.data.value]
+    );
+
+    const handler = {
+      has: function(target, property) {
+        return property[property.length - 1] !== "_" ? true : property.slice(0, -1) in target;
+      },
+      get: function(target, property) {
+        if (property[property.length - 1] === "_") {
+          const newName = property[property.length - 1] === "_" ? property.slice(0, -1) : name;
+          return newName in target && target[newName] > 0 ? target[newName] : 0;
+        } else {
+          return property in target ? target[property] : game.settings.get("twodsix", "modifierForZeroCharacteristic");
```

Comment:
Do you need to also check for JOAT here?

Reply:
Fixed!